source/includes/_questions.md

```diff
@@ -38,7 +38,8 @@ curl "https://app.sixpark.com.au/api/v1/questionnaire/questions"
     "..."
   ],
   "links": {
-    "self": "https://app.sixpark.com.au/api/v1/questionnaire/questions"
+    "self": "https://app.sixpark.com.au/api/v1/questionnaire/questions",
+    "result": "https://app.sixpark.com.au/api/v1/questionnaire/results/:id"
```

Comment:
I think this is `results`,  but I can change it as part of the performance PR. Do we want singular or plural?

Reply:
hmmm plural